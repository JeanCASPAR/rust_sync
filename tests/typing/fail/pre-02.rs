use rust_sync::sync;

sync! {
    node oui() = (a)
    where
        a : int = pre 0 -> 1;
}

fn main() {}
