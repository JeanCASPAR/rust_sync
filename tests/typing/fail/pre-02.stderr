error: this expression will cause an access to an undefined value
 --> tests/typing/fail/pre-02.rs:8:19
  |
8 |         a : int = pre 0 -> 1;
  |                   ^^^^^
